src/managed_data/api.py

```diff
@@ -145,6 +226,101 @@ def get_cell_data(
     return data[column_name]
 
 
+def set_cell_data(
+    value: Any,
+    *,
+    database_id: str,
+    row_id: str,
+    column_id: str,
+    client: Optional[Client] = None,
+) -> Any:
+    """set data in a cell to some value.
+
+    uses the EnsureRows API endpoint"""
+
+    # first, get the type of the column
+    response = _api.describe_row(database_id, client=client)
+
+    column = [
+        col
+        for col in response["cols"]
+        if col["id"] == column_id or col["name"] == column_id or col["key"] == column_id
+    ]
+
+    if len(column) != 1:
+        raise DeepOriginException(
+            f"Could not find column {column_id} in database {database_id}"
+        )
+
+    column = column[0]
+    if column["type"] == "select":
+        if isinstance(value, list):
+            pass
+        elif isinstance(value, str):
+            value = [value]
+        elif value is None:
+            value = [""]
```

Comment:
passing an empty string unsets, but i think it's nice to be able to also pass None